apps/docs/preinstall.js

```diff
@@ -26,7 +26,7 @@ fs.readFile("./package.json", "utf8", function (err, data) {
     // Check if the package is in the @nextui-org namespace and has "workspace:*" as its version
     if (pkg.startsWith("@nextui-org/") && packageJson.dependencies[pkg] === "workspace:*") {
       // Get the latest version of the package under the specified tag
-      const latestVersion = execSync(`npm show ${pkg}@dev-v2 version`, {encoding: "utf8"}).trim();
+      const latestVersion = execSync(`npm show ${pkg}@canary version`, {encoding: "utf8"}).trim();
```

Comment:
maybe some packages doesn't have canary version yet. hence the doc build is failing.